code/modules/reagents/reactions/medical.dm

```diff
@@ -15,12 +15,6 @@
 	required_reagents = list(/datum/reagent/oxygen = 2, /datum/reagent/toxin/phoron = 0.1)
 	required_catalysts = list(/datum/reagent/toxin/phoron = 5)
 
-/datum/chemical_reaction/dermaline
-	name = "Dermaline"
-	results = list(/datum/reagent/medicine/dermaline = 3)
-	required_reagents = list(/datum/reagent/oxygen = 1, /datum/reagent/phosphorus = 1, /datum/reagent/medicine/kelotane = 1)
-	required_catalysts = list(/datum/reagent/consumable/larvajelly = 5)
```

Comment:
The jelly recipes don't consumes the jelly, the limoline ones do. Means each larva you grind is only gonna give you 15u of mera or derm. Intended?

Reply:
Intended. I'd like to keep both, but the duplicate recipes are an issue and we never get larba. Remind me in a few and I'll look at req efficiencies, as larva are uncommon now.